Modules/Core/Common/include/itkBSplineInterpolationWeightFunction.hxx

```diff
@@ -63,7 +61,9 @@ BSplineInterpolationWeightFunction<TCoordRep, VSpaceDimension, VSplineOrder>::Pr
   Superclass::PrintSelf(os, indent);
 
   os << indent << "NumberOfWeights: " << Self::NumberOfWeights << std::endl;
-  os << indent << "SupportSize: " << m_SupportSize << std::endl;
+
+  constexpr auto supportSize = Self::SupportSize;
+  os << indent << "SupportSize: " << supportSize << std::endl;
```

Comment:
Why not print `Self::SupportSize` directly? Why print this constant at all?

Reply:
> Why not print Self::SupportSize directly?

Good question @dzenanz. Linux/GCC rejected the code when Self::SupportSize was directly inserted into the output stream (os). It did complain about an "undefined reference to itk::BSplineInterpolationWeightFunction::SupportSize'", at https://open.cdash.org/viewBuildError.php?buildid=7435820 While it was just fine to the other platforms/compilers.

> Why print this constant at all?

Honestly I don't know. The SupportSize of a BSplineInterpolationWeightFunction was always just filled by `VSplineOrder + 1` during initialization.  The original `m_SupportSize` could never be modified by the user either. Would you agree to just remove this print statement?

Reply:
Yes, removing the print statement is the best in my opinion.

Reply:
@dzenanz Just removed the print statement for SupportSize, with force-push https://github.com/InsightSoftwareConsortium/ITK/compare/eae5572cde584c2020fd0eb68035e45c2ab6acc5..4c58c6fa7f5b7f5eb0689de17b8e7e7613442144